pkg/util/time.go

```diff
@@ -48,6 +48,20 @@ func ParseTime(s string) (int64, error) {
 	return 0, httpgrpc.Errorf(http.StatusBadRequest, "cannot parse %q to a valid timestamp", s)
 }
 
+func ParseDurationMs(s string) (int64, error) {
```

Comment:
Can we add a unit test for this function? Especially it's exported now.

Reply:
will add unit test for this function